### DISCUSSION TITLE
После обновления перестало нормально работать

### DISCUSSION THREAD

**Discussion by Programator63** [bol-van/zapret#1011] [Q&A]
После обновления Zapret перестал запускаться. Удалив файл 10-inherit-nfqws, мне удалось запустить его снова, но теперь перестали открываться заблокированные сайт, иск ютуб. Роутер кинетик Extra - версия прошивка 4.2.4

Файлы в custom.d:
- 10-inherit-nfqws
- 50-discord

Конфиг:
[config.txt](https://github.com/user-attachments/files/18274828/config.txt)

Пробовал как-то восстановить работу, но не получилось.

**Comment by bol-van** [comment]
Рекомендую настроить с ноля. Удалить все, включая старый custom

**Reply by Programator63** [reply]
Переустановил и запустилось, но на последней версии вообще не хочет работать нормально, видео с ютуба не загружаются (пробовал старый конфиг, и искал новый)
